build-tasks/common.js

```diff
@@ -0,0 +1,141 @@
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+
+function readFile(fileName) {
+    try {
+        return fs.readFileSync(String(fileName), {encoding: 'utf8'});
+    } catch (e) {
+        console.error(`Unable to read file: ${String(fileName)}. Error code: ${e.code}`);
```

Comment:
Nit: is the `String(...)` necessary? Can we just do <code>Unable to read file: ${fileName}. ...`</code>?